.github/workflows/tests.yaml

```diff
@@ -22,17 +22,42 @@ jobs:
     name: Tests on Go ${{ matrix.go }}
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@v3.3.0
+        uses: actions/checkout@v4
         with:
           path: go/src/github.com/Workiva/go-datastructures
 
       - name: Setup Go
-        uses: actions/setup-go@v4
+        uses: actions/setup-go@v5.0.0
         with:
           go-version: ${{ matrix.go }}
 
+      - name: install go2xunit
+        run: |
+          git clone https://github.com/tebeka/go2xunit.git
+          cd go2xunit
+          git checkout v1.4.10
+          go install 
+          cd ..
```

Comment:
Can we just use `go install github.com/tebeka/go2xunit@v1.4.10`?

Reply:
No because that uses github credentials which we aren't supposed to use in an OSS repo

Reply:
I don't think we'd need to use the setup credentials step to do this, because go2xunit a public repo

Reply:
I see your failure from before -- that's unfortunate; seems like we should be able to fix it because all `go install` does is `git clone` and `go install` like you have here, but it's probably not worth the effort to try and fix it :shrug: